### ISSUE TITLE
Swiper JS bug in Elementor 2.9.2

### ISSUE THREAD

**Issue by steve231293** [elementor/elementor#10569]
## Description
Hello team,

![download](https://user-images.githubusercontent.com/7134267/74618016-99f77e00-5162-11ea-9591-61438dd851ca.png)

Look like frontend.js rewrite class Swiper,  So if there is any themes or plugins used swiper.js lib too, then it has js bug.

Please check & fix it.

## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

**Comment by steve231293** [comment]
Additional Infomation:
Init code: used option 'el' instead of param;
`var swiper = new Swiper({
      el: '.swiper-container',
      pagination: {
        el: '.swiper-pagination',
      },
    });`

`var swiper = new Swiper('.swiper-container', {
      el: '',
      pagination: {
        el: '.swiper-pagination',
      },
    });`

**Comment by steve231293** [comment]
I have an another problem with Elementor.

I change init code to param like above then it will can work with Elementor.
But Elementor replace all my swiper breakpoints. please just edit Elementor widgets & don't touch to Global Swiper.

Please check & fix it soon. Thanks

**Comment by shilo-ey** [comment]
Thanks for your feedback.

We will address this issue soon and let you know when we resolve it.

Thanks!

**Comment by mackyangeles** [comment]
I updated Elementor to 2.9.2 but still the breakpoint for swiper not showing properly. I want to show just 1 post but instead I’m getting the default. Any fix on this?

**Comment by steve231293** [comment]
Hello @shilo-ey,
Is there any news on this?

**Comment by NenadObradovic** [comment]
I fixed my breakpoints functionality with not good solution, since I had to do something about this problem.
 
Example solution:

```
var options = {
 // slider settings
};

var breakpoints = {
    320: {
	slidesPerView: 1,
    },
    769: {
	slidesPerView: 2,
    },
};

options.breakpoints = breakpoints;

var swiper = new Swiper($('.slider-selector'), options);
			
if ($('body').hasClass('elementor-page')) {
    // For dynamically functionality you need to create switch, if-else or some loop function for example
    swiper.params.slidesPerView = breakpoints['769'].slidesPerView; 

    swiper.params.breakpoints = breakpoints;
	
    swiper.update();
}
```

I hope plugin developers will fix this problem soon. My opinion is that is not a good practice to override global 3rd party script object and to messed other plugins/themes/elements.

**Comment by steve231293** [comment]
Hello NenadObradovic,
Thanks for sharing the code.

But your fixed will not working on window resize. All breakpoints will be replace again.

This is a big bug & huge affect to exist themes & plugins. Seem Elementor team didn't keep the eye to it.

**Comment by steve231293** [comment]
This bug fixed in Elementor 2.9.5

**Comment by shilo-ey** [comment]
Read more here: https://developers.elementor.com/swiper-update-to-5-3-6-new-wrapper-class/
